packages/tests/e2e/robot/tests/connection/security.robot

```diff
@@ -0,0 +1,32 @@
+*** Variables ***
+${ENCRYPTED_CONN_ID}         ecrypted-conn
+
+*** Settings ***
+Documentation       OdahuFlow's API operational check for operations on connection resources
+Test Timeout        20 minutes
+Resource            ../../resources/keywords.robot
+Variables           ../../load_variables_from_profiles.py    ${CLUSTER_PROFILE}
+Library             Collections
+Force Tags          connection  cli  api
+Suite Setup         Run keywords  Set Environment Variable  ODAHUFLOW_CONFIG  ${LOCAL_CONFIG}  AND
+...                               Login to the api and edge  ${SA_DATA_SCIENTIST}  AND
+...                               Cleanup resources
+Suite Teardown      Run keywords  Cleanup resources  AND
+...                               Remove File  ${LOCAL_CONFIG}
+
+*** Keywords ***
+Cleanup resources
+    Shell  odahuflowctl --verbose conn delete --id ${ENCRYPTED_CONN_ID} --ignore-not-found
+
+
+*** Test Cases ***
+Data scientist have not permissions to decrypt connection
```

Comment:
You duplicated this test two times here and in the conn_cli file. Is there any difference between them?

Reply:
removed. Forgot about this file